src/panels/lovelace/cards/hui-conditional-card.ts

```diff
@@ -75,6 +64,12 @@ class HuiConditionalCard extends HTMLElement implements LovelaceCard {
   public getCardSize() {
     return computeCardSize(this._card!);
   }
+
+  private update() {
```

Comment:
This is silly. Move EVERYTHING from the hass setter, except for assigning `this._hass = hass`, into this method. So that the update method is responsible for updates.

Reply:
Just like `updateElements` for conditional element

Reply:
Yes I forgot picture-elements. Done.